feg/gateway/docker/docker-compose.yml

```diff
@@ -148,10 +162,16 @@ services:
              /usr/bin/redis-cli shutdown"
 
   td-agent-bit:
-    <<: *pyservice
+    <<: *pyservice_base
     container_name: td-agent-bit
     logging:
       driver: json-file
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://127.0.0.1:2020"]
```

Comment:
what is this 2020 port? is this something standard from td-agent-bit
Why do we need to add healthcheck on this service now?

Reply:
This container starts with the "generate config" step before service starts and starts processing incoming messages.
The rest of the services rely on the service that can consume messages. So we check when service in the container is started and ready to handle incoming messages so then we can start the rest of the containers.